test/lib/notifier_test.rb

```diff
@@ -0,0 +1,66 @@
+require 'test_helper'
+
+class NotifierTest < Minitest::Test
+  def setup
+    @bukkit               = ENV.fetch "EXPORT_BUCKET"
+  end
+
+  def test_export_date
+    export_date = ENV.fetch "EXPORT_DATE"
+    assert_equal export_date, AuroraBootstrapper::Notifier.new(s3_path: @bukkit).export_date
```

Comment:
should we test the override logic here? It's a bit involved and it would be nice to have the peace of mind that it's covered in case we ever need to make changes

Reply:
yup no surprises